### DISCUSSION TITLE
Resurrect Classic Details Pane from Win7

### DISCUSSION THREAD

**Discussion by vippium** [valinet/ExplorerPatcher#1282] [Ideas]
Can EP work on resurrecting the Classic details pane from win7. Currently, there is only 1 way to make this happen is to use OldNewExplorer with EP. But it would be nice to have it in ExplorerPatcher. With that details pane, we can use both Preview Pane and Details Pane together at same time. Like this : -

**Without OldNewExplorer :**
![2023-04-09_18h48_58](https://user-images.githubusercontent.com/93650504/230779167-49758231-d161-414f-9878-be035500bc56.png)

**With OldNewExplorer :**
![2023-04-09_20h10_59](https://user-images.githubusercontent.com/93650504/230779299-9c69c56d-f93a-424c-aa96-f189c0bec01c.png)

**Comment by pyrates999** [comment]
I am unable to find the source code of old new explorer and the last release was in 2013.

**Reply by vippium** [reply]
If you do, can I try for the Source Code of it..

**Reply by vippium** [reply]
> I am unable to find the source code of old new explorer and the last release was in 2013.

You can try to ask admin on his forum of SAB...Just request him and tell the reason as he is the dev of both SAB and O.N.E.

**Comment by pyrates999** [comment]
If you can find the source code of OldNewExplorer, go for it.

**Reply by vippium** [reply]
https://msfn.org/board/forum/181-startisback/

Forum Link

**Reply by vippium** [reply]
i'm on the way for the source code...just talking with tihly (SAB developer) for that.

**Comment by Ben2557** [comment]
I already made a post about this : https://github.com/valinet/ExplorerPatcher/discussions/1130
